support/demoPageSync.js

```diff
@@ -1,24 +1,35 @@
 const { close, copyFileSync, open, utimes, watch } = require("fs");
 const { normalize } = require("path");
+const chokidar = require('chokidar');
 
 const projectRoot = normalize(`${__dirname}/../`);
-const sourceDir = `${projectRoot}src/`;
-const sourceDemoDir = `${sourceDir}demos/`;
-const destinationDir = `${projectRoot}www/`;
-const destinationDemoDir = `${destinationDir}demos/`;
-const sourceIndexFile = `${sourceDir}index.html`;
-const buildTriggeringFile = `${sourceDir}components/interfaces.ts`;
+const sourceDir = normalize(`${projectRoot}src/`);
```

Comment:
I needed to normalize the directories used in the replace function, since the strings had different slashes (mix of forward and backward). I supposed we could remove it from `sourceIndexFile` and `buildTriggeringFile`. Want me to?

Reply:
Gotcha. Nah, we can leave it.